framework/src/st_things/things_stack/framework/things_security_manager.c

```diff
@@ -718,7 +718,7 @@ static OicSecKey_t primary_cert;
 static OicSecKey_t primary_key;
 
 //Samsung_OCF_RootCA.der
-const unsigned char g_regional_root_ca[] = {
```

Comment:
how abut `const unsigned char * const`

Reply:
No need to make it 'const unsigned char * const'. I will revert back my changes.